readme.md

```diff
@@ -130,6 +130,10 @@ Type: [`stream.Readable`](https://nodejs.org/api/stream.html#stream_class_stream
 - [`webp`](https://en.wikipedia.org/wiki/WebP)
 - [`flif`](https://en.wikipedia.org/wiki/Free_Lossless_Image_Format)
 - [`cr2`](https://fileinfo.com/extension/cr2)
+- [`orf`](https://en.wikipedia.org/wiki/ORF_format)
+- [`arw`](https://en.wikipedia.org/wiki/Raw_image_format#ARW)
+- [`dng`](https://en.wikipedia.org/wiki/Digital_Negative)
+- [`nef`](https://en.wikipedia.org/wiki/Raw_image_format)
```

Comment:
Add the full format names after the link, like done on some of the other entries.

Reply:
Done.